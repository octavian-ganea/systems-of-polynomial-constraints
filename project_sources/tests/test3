x^2 + y^2 + z^2 = 0
x - 1 <= 0