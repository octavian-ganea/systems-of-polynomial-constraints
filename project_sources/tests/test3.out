Parsing file test3
Parsing succeded. Results:
List of all variables:
Vector(x, y, z)
List of all polynomials:
Vector((1.0 y^2 + (-0.01  + 1.0 z^2)) + 1.0 x^2, -1.0  + 1.0 x)

List of desired signs : Vector(=, <=)
Analysed polynomials: 
(1.0 y^2 + (-0.01  + 1.0 z^2)) + 1.0 x^2
-1.0  + 1.0 x
vars: Vector(x, y, z)

Univariate projection (on real axis) contains these polynomials: 
-0.01  + 1.0 z^2
-0.04  + 4.0 z^2
0.99  + 1.0 z^2
-2.56  + 256.0 z^2
3.96  + 4.0 z^2
0.0016  + -0.32 z^2 + 16.0 z^4
1.0E-4  + -0.02 z^2 + 1.0 z^4
0.9801  + 1.98 z^2 + 1.0 z^4

There are 129 cells :
Map(x -> 0.0, y -> 0.0, z -> -1.1000050003350859)
Map(x -> 1.0, y -> 0.0, z -> -1.1000050003350859)
Map(x -> 2.0, y -> 0.0, z -> -1.1000050003350859)
Map(x -> 0.0, y -> 0.0, z -> -0.10000500033508586)
Map(x -> 1.0, y -> 0.0, z -> -0.10000500033508586)
Map(x -> 2.0, y -> 0.0, z -> -0.10000500033508586)
Map(x -> 0.0, y -> 0.0, z -> -0.10000312524615036)
Map(x -> 1.0, y -> 0.0, z -> -0.10000312524615036)
Map(x -> 2.0, y -> 0.0, z -> -0.10000312524615036)
Map(x -> 0.0, y -> 0.0, z -> -0.10000125015721487)
Map(x -> 1.0, y -> 0.0, z -> -0.10000125015721487)
Map(x -> 2.0, y -> 0.0, z -> -0.10000125015721487)
Map(x -> 0.0, y -> 0.0, z -> -0.10000062519537876)
Map(x -> 1.0, y -> 0.0, z -> -0.10000062519537876)
Map(x -> 2.0, y -> 0.0, z -> -0.10000062519537876)
Map(x -> 0.0, y -> 0.0, z -> -0.10000000023354266)
Map(x -> 1.0, y -> 0.0, z -> -0.10000000023354266)
Map(x -> 2.0, y -> 0.0, z -> -0.10000000023354266)
Map(x -> 0.0, y -> -1.1000000002335426, z -> -2.567535073438876E-10)
Map(x -> 1.0, y -> -1.1000000002335426, z -> -2.567535073438876E-10)
Map(x -> 2.0, y -> -1.1000000002335426, z -> -2.567535073438876E-10)
Map(x -> 0.0, y -> -0.10000000023354266, z -> -2.567535073438876E-10)
Map(x -> 1.0, y -> -0.10000000023354266, z -> -2.567535073438876E-10)
Map(x -> 2.0, y -> -0.10000000023354266, z -> -2.567535073438876E-10)
Map(x -> -1.1000000002335426, y -> -1.1677133565646258E-10, z -> -2.567535073438876E-10)
Map(x -> -0.10000000023354266, y -> -1.1677133565646258E-10, z -> -2.567535073438876E-10)
Map(x -> -1.1677133565646258E-10, y -> -1.1677133565646258E-10, z -> -2.567535073438876E-10)
Map(x -> 0.09999999999999999, y -> -1.1677133565646258E-10, z -> -2.567535073438876E-10)
Map(x -> 0.55, y -> -1.1677133565646258E-10, z -> -2.567535073438876E-10)
Map(x -> 1.0, y -> -1.1677133565646258E-10, z -> -2.567535073438876E-10)
Map(x -> 2.0, y -> -1.1677133565646258E-10, z -> -2.567535073438876E-10)
Map(x -> -1.0000010008260192, y -> 0.09999999999999999, z -> -2.567535073438876E-10)
Map(x -> -1.0008260191095398E-6, y -> 0.09999999999999999, z -> -2.567535073438876E-10)
Map(x -> 0.49999949958699047, y -> 0.09999999999999999, z -> -2.567535073438876E-10)
Map(x -> 1.0, y -> 0.09999999999999999, z -> -2.567535073438876E-10)
Map(x -> 2.0, y -> 0.09999999999999999, z -> -2.567535073438876E-10)
Map(x -> 0.0, y -> 1.1, z -> -2.567535073438876E-10)
Map(x -> 1.0, y -> 1.1, z -> -2.567535073438876E-10)
Map(x -> 2.0, y -> 1.1, z -> -2.567535073438876E-10)
Map(x -> 0.0, y -> -1.0000075498039194, z -> 0.09999999972003565)
Map(x -> 1.0, y -> -1.0000075498039194, z -> 0.09999999972003565)
Map(x -> 2.0, y -> -1.0000075498039194, z -> 0.09999999972003565)
Map(x -> 0.0, y -> -7.549803919311563E-6, z -> 0.09999999972003565)
Map(x -> 1.0, y -> -7.549803919311563E-6, z -> 0.09999999972003565)
Map(x -> 2.0, y -> -7.549803919311563E-6, z -> 0.09999999972003565)
Map(x -> -1.0000006060479008, y -> -7.525017629407879E-6, z -> 0.09999999972003565)
Map(x -> -6.060479006941411E-7, y -> -7.525017629407879E-6, z -> 0.09999999972003565)
Map(x -> 0.4999996969760497, y -> -7.525017629407879E-6, z -> 0.09999999972003565)
Map(x -> 1.0, y -> -7.525017629407879E-6, z -> 0.09999999972003565)
Map(x -> 2.0, y -> -7.525017629407879E-6, z -> 0.09999999972003565)
Map(x -> -1.0000008602201074, y -> -7.500231339504195E-6, z -> 0.09999999972003565)
Map(x -> -8.60220107260156E-7, y -> -7.500231339504195E-6, z -> 0.09999999972003565)
Map(x -> 0.49999956988994637, y -> -7.500231339504195E-6, z -> 0.09999999972003565)
Map(x -> 1.0, y -> -7.500231339504195E-6, z -> 0.09999999972003565)
Map(x -> 2.0, y -> -7.500231339504195E-6, z -> 0.09999999972003565)
Map(x -> -1.0000075498039194, y -> -8.696485045952252E-9, z -> 0.09999999972003565)
Map(x -> -7.549803919311563E-6, y -> -8.696485045952252E-9, z -> 0.09999999972003565)
Map(x -> -3.3485301693966595E-8, y -> -8.696485045952252E-9, z -> 0.09999999972003565)
Map(x -> 7.48283331592363E-6, y -> -8.696485045952252E-9, z -> 0.09999999972003565)
Map(x -> 0.5000037414166579, y -> -8.696485045952252E-9, z -> 0.09999999972003565)
Map(x -> 1.0, y -> -8.696485045952252E-9, z -> 0.09999999972003565)
Map(x -> 2.0, y -> -8.696485045952252E-9, z -> 0.09999999972003565)
Map(x -> -1.0000010008260192, y -> 7.48283836941229E-6, z -> 0.09999999972003565)
Map(x -> -1.0008260191095398E-6, y -> 7.48283836941229E-6, z -> 0.09999999972003565)
Map(x -> 0.49999949958699047, y -> 7.48283836941229E-6, z -> 0.09999999972003565)
Map(x -> 1.0, y -> 7.48283836941229E-6, z -> 0.09999999972003565)
Map(x -> 2.0, y -> 7.48283836941229E-6, z -> 0.09999999972003565)
Map(x -> 0.0, y -> 1.0000074828383694, z -> 0.09999999972003565)
Map(x -> 1.0, y -> 1.0000074828383694, z -> 0.09999999972003565)
Map(x -> 2.0, y -> 1.0000074828383694, z -> 0.09999999972003565)
Map(x -> 0.0, y -> -1.0000053848394372, z -> 0.09999999986001781)
Map(x -> 1.0, y -> -1.0000053848394372, z -> 0.09999999986001781)
Map(x -> 2.0, y -> -1.0000053848394372, z -> 0.09999999986001781)
Map(x -> 0.0, y -> -5.3848394372129995E-6, z -> 0.09999999986001781)
Map(x -> 1.0, y -> -5.3848394372129995E-6, z -> 0.09999999986001781)
Map(x -> 2.0, y -> -5.3848394372129995E-6, z -> 0.09999999986001781)
Map(x -> -1.0000006105544998, y -> -5.350138631348344E-6, z -> 0.09999999986001781)
Map(x -> -6.105544998352905E-7, y -> -5.350138631348344E-6, z -> 0.09999999986001781)
Map(x -> 0.4999996947227501, y -> -5.350138631348344E-6, z -> 0.09999999986001781)
Map(x -> 1.0, y -> -5.350138631348344E-6, z -> 0.09999999986001781)
Map(x -> 2.0, y -> -5.350138631348344E-6, z -> 0.09999999986001781)
Map(x -> -1.0000008620227472, y -> -5.3154378254836874E-6, z -> 0.09999999986001781)
Map(x -> -8.620227472226913E-7, y -> -5.3154378254836874E-6, z -> 0.09999999986001781)
Map(x -> 0.4999995689886264, y -> -5.3154378254836874E-6, z -> 0.09999999986001781)
Map(x -> 1.0, y -> -5.3154378254836874E-6, z -> 0.09999999986001781)
Map(x -> 2.0, y -> -5.3154378254836874E-6, z -> 0.09999999986001781)
Map(x -> -1.0000053848394372, y -> -1.213595401148708E-8, z -> 0.09999999986001781)
Map(x -> -5.384839437185667E-6, y -> -1.213595401148708E-8, z -> 0.09999999986001781)
Map(x -> -4.684371870556415E-8, y -> -1.213595401148708E-8, z -> 0.09999999986001781)
Map(x -> 5.291151999774539E-6, y -> -1.213595401148708E-8, z -> 0.09999999986001781)
Map(x -> 0.5000026455759999, y -> -1.213595401148708E-8, z -> 0.09999999986001781)
Map(x -> 1.0, y -> -1.213595401148708E-8, z -> 0.09999999986001781)
Map(x -> 2.0, y -> -1.213595401148708E-8, z -> 0.09999999986001781)
Map(x -> -1.0000010008260192, y -> 5.291165917460713E-6, z -> 0.09999999986001781)
Map(x -> -1.0008260191095398E-6, y -> 5.291165917460713E-6, z -> 0.09999999986001781)
Map(x -> 0.49999949958699047, y -> 5.291165917460713E-6, z -> 0.09999999986001781)
Map(x -> 1.0, y -> 5.291165917460713E-6, z -> 0.09999999986001781)
Map(x -> 2.0, y -> 5.291165917460713E-6, z -> 0.09999999986001781)
Map(x -> 0.0, y -> 1.0000052911659174, z -> 0.09999999986001781)
Map(x -> 1.0, y -> 1.0000052911659174, z -> 0.09999999986001781)
Map(x -> 2.0, y -> 1.0000052911659174, z -> 0.09999999986001781)
Map(x -> 0.0, y -> -1.0000010008260192, z -> 0.09999999999999999)
Map(x -> 1.0, y -> -1.0000010008260192, z -> 0.09999999999999999)
Map(x -> 2.0, y -> -1.0000010008260192, z -> 0.09999999999999999)
Map(x -> 0.0, y -> -1.0008260191095398E-6, z -> 0.09999999999999999)
Map(x -> 1.0, y -> -1.0008260191095398E-6, z -> 0.09999999999999999)
Map(x -> 2.0, y -> -1.0008260191095398E-6, z -> 0.09999999999999999)
Map(x -> -1.0000006610284113, y -> -7.507098209977795E-7, z -> 0.09999999999999999)
Map(x -> -6.610284113257049E-7, y -> -7.507098209977795E-7, z -> 0.09999999999999999)
Map(x -> 0.49999966948579433, y -> -7.507098209977795E-7, z -> 0.09999999999999999)
Map(x -> 1.0, y -> -7.507098209977795E-7, z -> 0.09999999999999999)
Map(x -> 2.0, y -> -7.507098209977795E-7, z -> 0.09999999999999999)
Map(x -> -1.0000008665293467, y -> -5.005936228860193E-7, z -> 0.09999999999999999)
Map(x -> -8.665293466422452E-7, y -> -5.005936228860193E-7, z -> 0.09999999999999999)
Map(x -> 0.4999995667353267, y -> -5.005936228860193E-7, z -> 0.09999999999999999)
Map(x -> 1.0, y -> -5.005936228860193E-7, z -> 0.09999999999999999)
Map(x -> 2.0, y -> -5.005936228860193E-7, z -> 0.09999999999999999)
Map(x -> 0.0, y -> 0.9999994994063771, z -> 0.09999999999999999)
Map(x -> 1.0, y -> 0.9999994994063771, z -> 0.09999999999999999)
Map(x -> 2.0, y -> 0.9999994994063771, z -> 0.09999999999999999)
Map(x -> 0.0, y -> 0.0, z -> 0.10000000008950266)
Map(x -> 1.0, y -> 0.0, z -> 0.10000000008950266)
Map(x -> 2.0, y -> 0.0, z -> 0.10000000008950266)
Map(x -> 0.0, y -> 0.0, z -> 0.10000000017900534)
Map(x -> 1.0, y -> 0.0, z -> 0.10000000017900534)
Map(x -> 2.0, y -> 0.0, z -> 0.10000000017900534)
Map(x -> 0.0, y -> 0.0, z -> 1.1000000001790053)
Map(x -> 1.0, y -> 0.0, z -> 1.1000000001790053)
Map(x -> 2.0, y -> 0.0, z -> 1.1000000001790053)
====== FINAL RESULT FOR test3 ================================
Map(x -> 0.09999999999999999, y -> -1.1677133565646258E-10, z -> -2.567535073438876E-10)
====================================================

